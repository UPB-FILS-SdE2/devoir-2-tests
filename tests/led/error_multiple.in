led on 5 6
led on 0 0
led brightness set 100 0 0
exit