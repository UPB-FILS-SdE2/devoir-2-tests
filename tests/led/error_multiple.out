cmd: Invalid LED.
cmd: Set pixel 0 0 to 9
cmd: Invalid brightness.
cmd: 